app/models/image.rb

```diff
@@ -0,0 +1,10 @@
+class Image
+  include ActiveModel::Serialization
```

Comment:
As we have an ImageSerializer, I don't think this line is required. This class is just a property holder

Reply:
It is required, if you do not put it you get

```
 NoMethodError: undefined method `read_attribute_for_serialization' 
```